### PR TITLE
Fixes by the issues

### PR DESCRIPTION
**Pull request by irina060981**
For the https://github.com/alpheios-project/alpheios_nemo_ui/issues/119
Fixed the text and styles:

**Desktop**
image

**Mobile**
image

### CODE REVIEW

**Comment by irina060981** [comment]
For the issue https://github.com/alpheios-project/alpheios_nemo_ui/pull/115
fixed color for the logged in text
image

**Comment by irina060981** [comment]
Fixed styles for main-navbar after removing search input

**Comment by irina060981** [comment]
For the issue https://github.com/alpheios-project/alpheios_nemo_ui/issues/114

I suggest to add logo to the menu (because there is very less place in the header on mobile)
image

**Comment by irina060981** [comment]
For the issue https://github.com/alpheios-project/alpheios_nemo_ui/issues/116
As search input was hidden - i simply fixed styles
image

**Comment by irina060981** [comment]
For the issue
https://github.com/alpheios-project/alpheios_nemo_ui/issues/73

File: alpheios_nemo_ui/data/templates/alpheios/mainnavmenu.html [hunk 1]
```diff
@@ -1,42 +1,49 @@
...
+      </a>
+    </div>
+    <div class="main-dropdown-logout">
+        {% include "alpheios::logout.html" %}
+    </div>
+
+    <div class="main-dropdown-logo-bottom">
+        <a href="{{ url_for('.r_index') }}">
+          <img class="logo" src="{{url_for('.secondary_assets', filetype='static', asset='alpheios-logo.svg')}} " alt="Alpheios Reading Center" />
+          <h4>Reading Center</h4>
```
Comment (balmas) [inline alpheios_nemo_ui/data/templates/alpheios/mainnavmenu.html]: what if we just put the icon next to the alpheios.net link above?

Comment (irina060981) [reply alpheios_nemo_ui/data/templates/alpheios/mainnavmenu.html]: It would be like this
image

**Comment by irina060981** [comment]
I have added here - https://github.com/alpheios-project/alpheios_nemo_ui/issues/42
Hide alpheios-nemo-toolbar (i have used `visibility: hidden`, to avoid changing the layout)

**Comment by irina060981** [comment]
Also this issue is added too - https://github.com/alpheios-project/alpheios_nemo_ui/issues/118

**Pull request merged**